### PR TITLE
Make changes a mandatory field

### LINKED ISSUE #138: Indication of changes as a mandatory field
Herausforderung:
Aktuell muss man beim Reviewing von Bearbeitungen (insofern es keine Erst-Erstellung vom Inhalt ist), um die Änderung zur vorherigen Bearbeitung nachvollziehen zu können, in die Quelle des Inhaltes bzw. die vorherige Version mit der Vorschau vergleichen. Eigentlich sollten die Autoren bei ihren Bearbeitungen immer ihre Änderungen, Hinzufügen von Inhalt in den "Changes" kenntlich machen. Die meisten Autoren übersehen/machen das aber nicht. Das manuelle "Herausfinden" der Änderung kostet viel Zeit, obwohl manchmal nur ein Rechtschreibfehler ausgebessert wurde und der Reviewer kann den zeitlichen Aufwand den er für das Reviewen hat nicht abschätzen. 

Lösungsvorschlag:
Bei der Bearbeitung eines Inhaltes durch einen Autor ist die Angabe der Änderung (“changes”) ein Pflichtfeld. Gute Dokumentation wird so garantiert und der Reviewer einer Bearbeitung weiß, welchen Umfang die Änderung des Inhaltes hat. Er/Sie kann sich so die Zeit für das ""Herausfinden"" der Änderungen sparen und auch Inhalte, die nicht sein/ihr Wissens-Schwerpunkt sind reviewen, wenn es z.B. nur um eine Formatänderung geht.  In der Liste der ungerüften Bearbeitungen wird auch (eine Vorschau) der Änderung ("changes") angezeigt.

Variante:
Statt einem freien Text unter "changes" gibt es auch die Möglichkeit aus verschiedenen Vorschlägen auszuwählen, die einfach nur noch angeklickt werden müssen, z.B. "Rechtschreibfehler", "Bild einfügen", "Layout-Änderung", "Inhaltliche Erweiterung/Verbesserung".....etc.

### PR DESCRIPTION
**Pull request by Knorrke**
Fixes #138 

Maybe a follow-up issue:
We have no visual clue for mandatory fields in neither editor... There is a `title` attribute on the disabled "Save" Button, but it's not really obvious that you have to fill in the changes field.

### CODE REVIEW

File: packages/private/edtr-io/src/plugins/types/common.tsx [hunk 1]
```diff
@@ -191,7 +191,7 @@ export function Controls(props: OwnProps) {
             title={
               maySave()
                 ? undefined
-                : 'Du musst zuerst die Lizenzbedingungen akzeptieren'
+                : 'Du musst zuerst die Lizenzbedingungen akzeptieren und die Änderungen ausfüllen'
```
Comment (inyono) [inline packages/private/edtr-io/src/plugins/types/common.tsx]: Does this text make sense? Shouldn't this also be dependent on `props.license` and `props.changes`?

Comment (Knorrke) [reply packages/private/edtr-io/src/plugins/types/common.tsx]: Maybe. But what I don't want is something like "You have to accept the license first" -> after accepting license: "You have to fill in the changes field"...

Comment (inyono) [reply packages/private/edtr-io/src/plugins/types/common.tsx]: Yep, I agree with you on that. But we shouldn't tell the user that he has to accept the license when the content has no license. Or even better: use the same conditions as in `maySave` (i.e. the current text if the content has both elements and both are missing. But `You have to accept the license first` if the changes have already been filled out)

Comment (Knorrke) [reply packages/private/edtr-io/src/plugins/types/common.tsx]: Oh right, forgot that license or changes could not exist at all.

**Comment by yolo-bird** [comment]
> fields

I think the visual clue is also very important for the project concerning the semantic plugins. 

I'll keep you updated about how people cope with this right noe. Think we will find that out in U - Begleitung when reviewing.

**Pull request merged**